test/map.test.js

```diff
@@ -0,0 +1,11 @@
+import map from "../src/map.js";
+
+function square(n) {
+  return n * n;
+}
+
+describe("map", () => {
+  it("maps numbers to squared", () => {
+    expect(map([4, 8], square)).toIncludeAllMembers([16, 64]);
+  });
```

Comment:
Looks like `toBe` does not compare arrays correctly. Changed to https://jest-extended.jestcommunity.dev/docs/matchers/Array#toincludeallmembersmembers and the test passes

Reply:
Or should it be "toStrictEqual"?

Reply:
Could also be ok, but since we are using jest-extended lets prefer that. It also has a lot of matchers for object types